src/scss/App.scss

```diff
@@ -173,7 +193,198 @@ body {
     border-right: 6px solid transparent;
     margin: 15px;
   }
+} 
+
+///// Search Bar Styles /////
+
+.search-bar {
+  height: 45px;
+  background-color: $white;
+}
+
+///// Player Container Styles /////
+
+.players-container {
+  display: flex;
+  box-shadow: -1px 0px 1px $grey3;
+} 
+
+.players-list-container {
+  display: flex;
+  flex-wrap: wrap;
+  width: 72%;
```

Comment:
I read your description comment and I'm not a fan of using width percentages. It's better to give pixel values to the cards, have them 4 in a row static (no responsiveness) and have the details section width take up the remaining space.